twindle-cli/twitter-puppeteer/index.js

```diff
@@ -1,19 +1,59 @@
-const puppeteer = require('puppeteer');
-const getTweet = async (pageUrl) => {
-  const browser = await puppeteer.launch({ headless: true });
-  const page = await browser.newPage();
-  await page.goto(pageUrl, {
-    waitUntil: 'networkidle0',
-  });
-  const tweets = await page.evaluate(() => {
-    const tweetNodes = Array.from(
-      document.querySelectorAll('div[lang]:not([lang=""])')
-    );
-    const data = tweetNodes.map((node) => ({
-      tweet: node.innerText,
-    }));
-    return data;
-  });
-  return { data: tweets };
+const puppeteer = require("puppeteer");
+const { waitFor } = require("../utils/helpers");
+
+/**
+ * Get tweets(even older than 7 days) using puppeteer
+ * @param {string} tweetID
+ */
+const getTweet = async (tweetID) => {
+  const pageURL = `https://twitter.com/anyone/status/${tweetID}`;
+
+  try {
+    // Modify this variable to control the size of viewport
+    const factor = 0.1;
+    const height = Math.floor(2000 / factor);
+    const width = Math.floor(1700 / factor);
+
+    const browser = await puppeteer.launch({
+      headless: false,
```

Comment:
Can be set back to true?

Reply:
Definitely, but false for testing purposes